Source/EventFlow/Aggregates/IAggregateRoot.cs

```diff
@@ -35,8 +35,13 @@ public interface IAggregateRoot
         IEnumerable<IAggregateEvent> UncommittedEvents { get; }
         bool IsNew { get; }
 
-        Task<IReadOnlyCollection<IDomainEvent>> CommitAsync(IEventStore eventStore, CancellationToken cancellationToken);
+        Task<IReadOnlyCollection<IDomainEvent>> CommitAsync(
+            IEventStore eventStore,
+            ISourceId sourceId,
```

Comment:
@MartinHave I know. The change is to a "internal" service not typically used by developers, but I added the change to the release note along with a description on how to get a new ID